src/alitra/frame_dataclasses.py

```diff
@@ -103,17 +103,17 @@ def from_array(
 
 @dataclass
 class Euler:
-    """Euler angles where (psi,phi,theta) is rotation about the z-,y-, and x- axis respectively. as_array and from_array
-    are both ordered by rotation about z,y,x. That is [psi,phi,theta]"""
+    """Euler angles where (psi,theta,phi) is rotation about the z-,y-, and x- axis respectively. as_array and from_array
+    are both ordered by rotation about z,y,x. That is [psi,theta,phi]"""
```

Comment:
This file needs to be reformated with Black. There is a bug in the pipeline where it doesn't fail on Black not having been run.